README.FreeBSD

```diff
@@ -16,22 +16,26 @@ git submodule update --init --recursive
 cmake .. -DCMAKE_BUILD_TYPE=RelWithDebInfo
 gmake
 
-3. Set up python 2.7
+3. Set up python 3.8
 -------------
 
 Install testing dependences either from packages or from pip.
 
 3.1. From packages:
 -------------
-pkg install python27 py27-yaml py27-daemon py27-msgpack
+pkg install python38 py38-yaml py38-six py38-gevent
```

Comment:
I would comment it in the commit message as well: python-daemon is not used anymore (you can link https://github.com/tarantool/test-run/commit/8797cb15bf34f8c70519221b3b9c78f08696aba6 and https://github.com/tarantool/tarantool/commit/130cf4659bf938d1fa1bf0e936b0a43a53fa8dcb here), python-msgpack is enabled in test-run as a submodule, while python-gevent and python-yaml are actually used in test-run.

Reply:
comment updated.